scripts/remove_past_intervals.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+# Program:
+# 		  User can remove pg's past intervals for a specified OSD
```

Comment:
Remove all pgs' past_intervals for a specified OSD.
Please use with caution as there is no turning back.